<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QWidget" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>148</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>49</red>
        <green>55</green>
        <blue>57</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>49</red>
        <green>55</green>
        <blue>57</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Mainwindow</string>
  </property>
  <widget class="QSpinBox" name="mySpinBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="suffix">
    <string>km/h</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="myLcdNumber">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>46</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QSlider" name="mySlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>miles/h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="myClearButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>108</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="myQuitButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>myQuitButton</sender>
   <signal>clicked()</signal>
   <receiver>Mainwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
